src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -149,12 +149,16 @@ private function registerAnnotationLoaders(ContainerBuilder $container)
     private function registerFileLoaders(ContainerBuilder $container)
     {
         $paths = [];
+        $globOptions = GLOB_BRACE | GLOB_NOSORT;
```

Comment:
I'm 👎 to add an `@internal` class. Keep it as is or add a dependency to the Symfony Finder but w e don't want to reinvent the well.

Btw, the dependency to `symfony/finder` will be a soft dependency as it is only used in the bundle. And almost all people installing the SF bundle will have this dependency already installed as the finder is required by the SF FrameworkBundle.